src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -964,6 +912,7 @@ class NPlusOneDBSpanDetector(PerformanceDetector):
         "n_spans",
     )
 
+    type: DetectorType = DetectorType.N_PLUS_ONE_DB_QUERIES
```

Comment:
I think listing the types here is redundant as its part of the base class.

Reply:
Unfortunately not, it infers the type as `Literal[DetectorType.SLOW_SPAN]`